src/lib/prof-lean/crypto-hash.h

```diff
@@ -159,6 +159,10 @@ crypto_hash_self_test
   int verbose
 );
 
+#if defined(__cplusplus)
```

Comment:
This if defined(__cplusplus) should be deleted. It is an unmatched duplicate inside an if 0. The whole if 0 can be deleted if you choose.

Reply:
@jmellorcrummey Done